app/utils/PostcodeHelper.scala

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import uk.gov.hmrc.address.uk.Postcode
+
+object PostcodeHelper {
+
+  def displayPostcode(p:Option[String]) = p.fold("")(pc => Postcode.cleanupPostcode(pc).fold("")(_.toString))
```

Comment:
I didn't realise until this morning how many different patterns you could get out of fold!  

Would the equivalent be something like this? 
```scala
   def displayPostcode(p:Option[String]) = p.flatMap(Postcode.cleanupPostcode).map(_.toString).getOrElse("")
```

Reply:
done